### PR TITLE
Adjust `user-select` properties vlue.

### PR DESCRIPTION
**Pull request by k-utsumi**
Since there is no `auto` value, I added and adjut the following as a reference.
https://developer.mozilla.org/docs/Web/CSS/user-select

### CODE REVIEW

File: src/css/Properties.js [hunk 1]
```diff
@@ -460,7 +460,7 @@ var Properties = module.exports = {
     // U
     "unicode-bidi"                      : "normal | embed | isolate | bidi-override | isolate-override | plaintext",
     "user-modify"                       : "read-only | read-write | write-only",
-    "user-select"                       : "none | text | toggle | element | elements | all",
+    "user-select"                       : "none | auto | text | all | contain | element | elements | toggle",
```
Comment (nschonni) [inline src/css/Properties.js]: I don't see the `toggle` listed, where is that one coming from?

Comment (k-utsumi) [inline src/css/Properties.js]: From here?
https://www.w3.org/TR/2000/WD-css3-userint-20000216#user-select

Comment (frvge) [inline src/css/Properties.js]: That draft has been superseded. The current draft is https://drafts.csswg.org/css-ui-4/#propdef-user-select .

I can't find `toggle` in any of the more current drafts.

Comment (k-utsumi) [inline src/css/Properties.js]: I only wanted to use the `all` property.

I fixed that.
Thank you for feedback.

**Pull request merged**